modules/installation-aws-user-infra-bootstrap.adoc

```diff
@@ -4,26 +4,26 @@
 // * installing/installing_aws/installing-restricted-networks-aws.adoc
 
 [id="installation-aws-user-infra-bootstrap_{context}"]
-= Initializing the bootstrap node on AWS with user-provisioned infrastructure
+= Initializing the bootstrap sequence on AWS with user-provisioned infrastructure
 
 After you create all of the required infrastructure in Amazon Web Services (AWS),
-you can install the cluster.
+you can start the bootstrap sequence that initializes the {product-title} control plane.
 
 .Prerequisites
 
-* Configure an AWS account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VPC and associated subnets in AWS.
-* Create and configure DNS, load balancers, and listeners in AWS.
-* Create control plane and compute roles.
-* Create the bootstrap machine.
-* Create the control plane machines.
-* If you plan to manually manage the worker machines, create the worker machines.
+* You have configured an AWS account.
+* You have added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You have generated the Ignition config files for your cluster.
+* You have created and configured a VPC and associated subnets in AWS.
+* You have created and configured DNS, load balancers, and listeners in AWS.
+* You have created the security groups and roles required for your cluster in AWS.
+* You have created the bootstrap machine.
+* You have created the control plane machines.
+* You have created the worker nodes.
 
 .Procedure
 
-. Change to the directory that contains the installation program and run the
-following command:
+. Initialize the bootstrap process that initializes the {product-title} control plane:
```

Comment:
@kalexand-rh can you please expand on your suggestion here? I noticed that there are many cases where we use the `./openshift-install` command in this assembly but do not state that there is a need to run it from the directory where it is contained.

Reply:
It's about half and half, which is worse than I remember. After our last chat about this, I thought you wanted to strip the `./`. It might be worth making them all consistent.

Reply:
@kalexand-rh I have now sent a poll so that we can review this as a team.

Reply:
@kalexand-rh in the poll, the majority said that they would prefer to start the command with `./` and mention in the preceding text that it needs to be run from the directory containing the binary. I will update each of the steps in this PR that reference `./openshift-install` to reflect this. Thanks for highlighting it!

Reply:
I have now updated each `./openshift-install`reference in the assemblies in the OCP on AWS installation path accordingly.